### PR TITLE
Fix a typo in API.md

### PR DESCRIPTION
**Pull request by d8660091**

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -749,7 +749,7 @@ setObservableConfig<Stream>({
 Observables in Recompose are plain objects that conform to the [ES Observable proposal](https://github.com/zenparsing/es-observable). Usually, you'll want to use them alongside an observable library like RxJS so that you have access to its suite of operators. By default, this requires you to convert the observables provided by Recompose before applying any transforms:
 
 ```js
-mapPropsStream($props => {
+mapPropsStream(props$ => {
   const $rxjsProps = Rx.Observable.from(props$)
   // ...now you can use map, filter, scan, etc.
   return $transformedProps
```
Comment (istarkov) [inline docs/API.md]: The same ;-) could you also please

File: docs/API.md [hunk 2]
```diff
@@ -749,7 +749,7 @@ setObservableConfig<Stream>({
 Observables in Recompose are plain objects that conform to the [ES Observable proposal](https://github.com/zenparsing/es-observable). Usually, you'll want to use them alongside an observable library like RxJS so that you have access to its suite of operators. By default, this requires you to convert the observables provided by Recompose before applying any transforms:
 
 ```js
-mapPropsStream($props => {
+mapPropsStream(props$ => {
   const $rxjsProps = Rx.Observable.from(props$)
```
Comment (d8660091) [inline docs/API.md]: Right, I can fix it.

File: docs/API.md [hunk 3]
```diff
@@ -749,8 +749,8 @@ setObservableConfig<Stream>({
 Observables in Recompose are plain objects that conform to the [ES Observable proposal](https://github.com/zenparsing/es-observable). Usually, you'll want to use them alongside an observable library like RxJS so that you have access to its suite of operators. By default, this requires you to convert the observables provided by Recompose before applying any transforms:
 
 ```js
-mapPropsStream($props => {
-  const $rxjsProps = Rx.Observable.from(props$)
+mapPropsStream(props$ => {
+  const rxjsProps$ = Rx.Observable.from(props$)
   // ...now you can use map, filter, scan, etc.
   return $transformedProps
```
Comment (istarkov) [inline docs/API.md]: The same for transformedProps as we use [Finnish notation](https://medium.com/@benlesh/observables-and-finnish-notation-df8356ed1c9b#.6fq7fovn7) ;-)

**Comment by istarkov** [comment]
Thank you! It's hard to see such typos.

**Comment by d8660091** [comment]
Thank you for having me :)

**Pull request merged**